src/sentry/receivers/onboarding.py

```diff
@@ -39,6 +41,12 @@
 
 logger = logging.getLogger("sentry")
 
+# Used to determine if we should or not record an analytic data
+# for a first event of a project with a minified stack trace
+START_DATE_TRACKING_FIRST_EVENT_WITH_MINIFIED_STACK_TRACE_PER_PROJ = datetime(
```

Comment:
@priscilawebdev this should probably be a server setting that exists in `server.py`

Reply:
I got feedback from Radu (if I remember well) that since this is only used in one single place, it should not be in `server.py` + `server.py`  is also mainly used to list variables with default values that users can modify in local installations

Agree with Radu's feedback and prefer to leave it here

Reply:
@priscilawebdev Correct, this would be something you might play with locally to validate functionality. But it doesn't matter that much to me cause we won't need this in the long term.

Reply:
In this case, lets leave it here :) soon it will be deleted